app/src/androidTest/java/com/github/ashutoshgngwr/noice/fragment/WakeUpTimerFragmentTest.kt

```diff
@@ -310,4 +309,18 @@ class WakeUpTimerFragmentTest {
     onView(withId(R.id.select_preset_button))
       .check(matches(withText("test-not-saved-preset-1")))
   }
+
+  @Test
+  fun testSetSchedule_ShowNotifySnackBar() {
+    every { Preset.readAllFromUserPreferences(any()) } returns arrayOf(
+      mockk(relaxed = true) {
+        every { id } returns "test-preset"
+        every { name } returns "test-preset"
+      }
+    )
+    fragmentScenario.recreate()
+
+    onView(withId(R.id.set_time_button)).perform(click())
+    onView(withSubstring("Alarm will go off")).check(matches(isDisplayed()))
+  }
```

Comment:
For this test-case to work, you need to mock WakeUpTimerManager.get() and return a timer in future.